packages/component_code_gen/README.md

```diff
@@ -0,0 +1,36 @@
+# AI Code Gen for Pipedream Components
+
+Generate components using OpenAI GPT-4.
+
+
+### Run
+
+```
+SCRIPT=generate_action.py
+APP=slack
+PROMPT="how to send myself a direct message?"
+poetry run python3 "$SCRIPT" --app "$APP" "$PROMPT"
+poetry run python3 "$SCRIPT" --app "$APP" "$PROMPT" --verbose # print debug logs
+```
+
+
+### Installation
```

Comment:
I would move this section up above the **Run** section since it's required to run the code.